Applications/AlexNet/jni/meson.build

```diff
@@ -0,0 +1,19 @@
+build_root = meson.build_root()
+res_path = meson.current_source_dir() / '..' / 'res'
+mnist_sources = [
+  'main.cpp',
+  'cifar_dataloader.cpp'
+]
+if get_option('platform') == 'tizen'
```

Comment:
There is no test with AlexNet  so we can remove it I guess?

Reply:
FIX, Also find typo : ```mnist_sources``` --> ```alex_sources```